tests/test_ironoak.py

```diff
@@ -1,17 +1,10 @@
 
-from ironoak.cli import main
-from ironoak.person import persondetector
-import unittest
+from ironoak.person import PersonDetector
+
+videoPath = "/Users/garyding/Public/github_gsoc/depthai-python/examples/models/construction_vest.mp4"
+det = PersonDetector.PersonDetector(videoPath)
+while True:
+    result = det.getResult()             #nonblock read
```

Comment:
Try using pytest for this instead of unittest. You would need to make a test function.